packages/web/src/scss/components/Grid/_tools.scss

```diff
@@ -7,14 +7,37 @@
 // * $gutters: the gutters map
 // * $breakpoints: the breakpoints map
 @mixin gutters($gutters, $breakpoints) {
+    // stylelint-disable custom-property-no-missing-var-function -- We are only building the fallback value here.
+    $column-property-name: --grid-spacing-x;
+    $row-property-name: --grid-spacing-y;
+    // stylelint-enable
+
+    $column-fallback: map.get($gutters, 'mobile');
```

Comment:
we still need to set the defaults for each breakpoint. Now you are using only the mobile values and the rest is not used. Please fix it :) 

See how it is broken in demos.